oci/manifest.go

```diff
@@ -28,7 +28,7 @@ import (
 	"kraftkit.sh/oci/handler"
 )
 
-type Image struct {
+type Manifest struct {
```

Comment:
I am not 100% sure I agree with this change because now `manifest.manifest` exists and can be confusing, but I'll not oppose it

Reply:
I guess it could be called `ImageManifest` -- the OCI spec calls it this, or just "manifest". I agree it is a bit confusing but they are in different packages.

See https://github.com/opencontainers/image-spec/blob/main/manifest.md